snap/README.md

```diff
@@ -27,24 +64,34 @@ You can see the current revisions available for your machine's architecture by r
 $ snap info edgexfoundry
 ```
 
-The snap can be installed using `snap install`. To install the snap from the edge channel:
+The snap can be installed using `snap install`. To install the latest stable version:
+
+```bash
+$ sudo snap install edgexfoundry
+```
+
+To install the snap from the edge channel:
 
 ```bash
 $ sudo snap install edgexfoundry --edge
 ```
 
-You can specify install specific releases using the `--channel` option. For example to install the Fuji release of the snap:
+**Note** - in general, installing from the edge channel is only recommended for development purposes. Depending on the state of the current development release, your mileage may vary.
+
+
+You can also specify specific releases using the `--channel` option. For example to install the Hanoi release of the snap:
 
 ```bash
-$ sudo snap install edgexfoundry --channel=fuji
+$ sudo snap install edgexfoundry --channel=hanoi
```

Comment:
I would instead mention `--channel=2.0` as that's the only way to install Ireland.